### DISCUSSION TITLE
/chat/completions endpoint

### DISCUSSION THREAD

**Discussion by rgstephens** [flyerhq/flutter_chat_ui#551] [Q&A] [answered]
With so many llm's implementing OpenAI's chat completions endpoint, is there an example implementation?

**Comment by demchenkoalex** [comment]
not atm, but I will make sure to add all the AI stuff in `v2`

**Comment by demchenkoalex** [comment, marked as answer]
example available in v2 https://github.com/flyerhq/flutter_chat_ui/issues/646 (Gemini, but idea is the same for anything)
